app/src/main/java/protect/card_locker/ManageGroupCursorAdapter.java

```diff
@@ -0,0 +1,365 @@
+package protect.card_locker;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.database.Cursor;
+import android.graphics.Color;
+import android.graphics.drawable.Drawable;
+import android.util.Log;
+import android.util.SparseBooleanArray;
+import android.util.TypedValue;
+import android.view.HapticFeedbackConstants;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import com.google.android.material.card.MaterialCardView;
+
+import java.math.BigDecimal;
+import java.text.DateFormat;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.core.graphics.BlendModeColorFilterCompat;
+import androidx.core.graphics.BlendModeCompat;
+import androidx.recyclerview.widget.RecyclerView;
+
+import protect.card_locker.preferences.Settings;
+
+public class ManageGroupCursorAdapter extends BaseCursorAdapter<ManageGroupCursorAdapter.ManageGroupListItemViewHolder> {
```

Comment:
This class seems to be a copy-paste of LoyaltyCardCursor. Please just use LoyaltyCardCursor instead

Reply:
I could have extended LoyaltyCardCursorAdapter with inheritance but, it was copied for this POC to kee modularity (when LoyaltyCardCursorAdapter changes, changes might not apply to here)

The same goes for content_main.xml and loyalty_card_layout

If you think to reusing the layout files and inherit LoyaltyCardCursorAdapter is better then I'll go for that